docs/src/pages/[platform]/connected-components/liveness/react-props.ts

```diff
@@ -78,13 +78,13 @@ export const FACE_LIVENESS_DETECTOR_CONFIG = [
   {
     name: `binaryPath?`,
     description:
-      'Overrides the WASM backend binary CDN path, the default is https://cdn.jsdelivr.net/npm/@tensorflow/tfjs-backend-wasm@3.11.0/dist/. When overriding this path ensure that the wasm version matches the version of @tensorflow/tfjs-backend-wasm installed by npm.',
+      'Overrides the WASM backend binary CDN path, the default is https://cdn.jsdelivr.net/npm/@tensorflow/tfjs-backend-wasm@3.11.0/dist/. When overriding this path ensure that the wasm version matches the version of @tensorflow/tfjs-backend-wasm installed by npm. We recommend overriding this path to be the Rekognition provided CDN at https://cdn.liveness.rekognition.aws.dev/face-detection/tensorflow/tfjs-backend-wasm/3.11.0/.',
```

Comment:
This is a wall of text, can we split this into multiple columns?

Reply:
Fixed - 
<img width="1236" alt="Screenshot 2023-06-14 at 3 00 45 PM" src="https://github.com/aws-amplify/amplify-ui/assets/68032955/caad89e1-0dd7-4ac3-a133-beb1360da026">